src/openforms/forms/tests/e2e_tests/test_registration_backend_conf.py

```diff
@@ -0,0 +1,184 @@
+from django.urls import resolve, reverse
+
+from asgiref.sync import sync_to_async
+from furl import furl
+
+from openforms.registrations.contrib.zgw_apis.tests.factories import (
+    ZGWApiGroupConfigFactory,
+)
+from openforms.tests.e2e.base import E2ETestCase, browser_page, create_superuser
+
+from ..factories import FormFactory
+from .test_form_designer import open_component_options_modal, phase
+
+
+class FormDesignerRegistrationBackendConfigTests(E2ETestCase):
+    async def test_configuring_zgw_api_group(self):
+        """
+        The flow in this test is:
+        - Configure the Registration backend to use set 1 of the configured ZGW apis
+        - Go to the file upload component and check that the query parameter in the request to the informatieobjecttype endpoint is the PK of the right group
+        - Go back to the registration tab and change which ZGW API group should be used
+        - Go to the file component and check that the query parameter in the request changes
+        """
```

Comment:
Docstrings are parsed in the test runner in verbosity mode, so for this test it would output `The flow in this test is:`, which isn't particularly useful. PEP-257 was written about docstrings, but the summary is to stick to the format:

```py
"""
Describe in imperative form the summary
{blank line}
Provide an elaborate description with all the necessary information, and
even ``RST``/Sphinx syntax.
"""
```